main.py

```diff
@@ -85,8 +94,145 @@ def recording_file(rec_file):
     run = subparsers.add_parser("run", description="Run the model on the given data")
     run.add_argument("saved_model", nargs=1, type=existing_model)
     run.add_argument("--output-file", "-o", nargs="?", default=None, type=str)
-    run.add_argument("recordings", nargs="+", type=recording_file)
+    run.add_argument("recording", nargs="+", type=recording_file)
 
     return parser.parse_args()
 
-# process_audio_directory
+
+def get_data_from_dir(data_dir, preprocess_method):
```

Comment:
Right now I save all extracted features under the directories `./data/processed/<accent>/<accent>-<feature>.npy`, where feature is either `mfcc` or `spectrogram`.

Reply:
ok sounds good I think that's what I've been using (might need to change fft to spectrogram though)